pykern/pkcli/test.py

```diff
@@ -18,6 +18,8 @@
 SUITE_D = "tests"
 
 _TEST_PY = re.compile(r"_test\.py$")
+_RUNTIME_WARNING_FAIL = r"RuntimeWarning: coroutine .+ was never awaited"
```

Comment:
`r` is unnecessary. The name is `_COROUTINE_NEVER_AWAITED`. It should be a `re.compile`.